ui/app/AppLayouts/Chat/ContactsColumn/ChannelList.qml

```diff
@@ -48,6 +48,65 @@ ScrollView {
         }
     }
 
+    PopupMenu {
+        property int channelIndex
+
+        id: channelContextMenu
+        width: 175
+
+        function openMenu(channelIndex) {
+            channelContextMenu.channelIndex = channelIndex
+            channelContextMenu.popup()
+        }
+
+        Action {
+            text: qsTr("View Group")
+            icon.source: "../../../img/group.svg"
+            icon.width: 13
+            icon.height: 13
+            onTriggered: console.log('TODO View group')
+        }
+        Action {
+            text: qsTr("Mute Chat")
+            icon.source: "../../../img/bell.svg"
+            icon.width: 13
+            icon.height: 13
+            onTriggered: console.log('TODO Mute')
```

Comment:
This should display a submenu with the following options: For 15 min, For 1 hour, For 8 hours, For 24 hours, Until I turn it back on